src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts

```diff
@@ -214,7 +214,10 @@ export class DeveloperYamlConfig extends LitElement {
         }
 
         .content {
-          padding: 28px 20px 16px;
```

Comment:
This one is still removed

Reply:
Oups, I forgot it. I used 16px as a base for padding, to keep the same spacing logic.